clang/lib/DPCT/APINames.inc

```diff
@@ -570,13 +631,109 @@ ENTRY(h2sin, h2sin, true, NO_FLAG, P4, "Successful", "NA")
 ENTRY(h2sqrt, h2sqrt, true, NO_FLAG, P4, "Successful", "NA")
 ENTRY(h2trunc, h2trunc, true, NO_FLAG, P4, "Successful", "NA")
 
+// 16 Bfloat16 Arithmetic Functions are overload with other ENTRY
+
+// 17 Bfloat162 Arithmetic Functions are overload with other ENTRY
+
+// 18 Bfloat16 Comparison Functions are overload with other ENTRY
+
+// 41 Bfloat162 Comparison Functions are overload with other ENTRY
```

Comment:
if this is useless code, please remove it

Reply:
I have optimized the description here.